### DISCUSSION TITLE
Welcome to DPsim Discussions!

### DISCUSSION THREAD

**Discussion by m-mirz** [sogno-platform/dpsim#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pipeacosta** [comment]
Dear Markus,

Thank you so much for this space. We are a team of students at Universidad Nacional de Colombia, working on a research project related to the energy transition in the 2030 horizon. We are very interested on using DPsim for evaluating power systems incorporating renewables and control systems.

**Comment by evpitts** [comment]
I'm part of a team that is working on developing an EMT simulation of the power grid for some control and optimization studies. We are very interested in using DPsim for the base of that simulation!

**Reply by stv0g** [reply]
Hi @evpitts, great to hear! We hope to establish DPsim as an open-source with more external contributors.
